cmake/developer_package/compile_flags/os_flags.cmake

```diff
@@ -430,9 +445,22 @@ else()
     # - https://gcc.gnu.org/onlinedocs/gcc/C_002b_002b-Dialect-Options.html
     # - https://gcc.gnu.org/onlinedocs/libstdc++/manual/abi.html
     if(CMAKE_COMPILER_IS_GNUCXX)
+        if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "8")
+            # Enable __FILE__ trim
+            ie_add_compiler_flags(-ffile-prefix-map=${OV_NATIVE_PROJECT_ROOT_DIR}/=)
+            ie_add_compiler_flags(-ffile-prefix-map=${OV_RELATIVE_BIN_PATH}/=)
```

Comment:
do we also need to use -fmacro-prefix-map or -fdebug-prefix-map ?

Reply:
specify this option will specify all `-f*-prefix-map`